app/scripts/lib/transaction/transaction-send-bundle.ts

```diff
@@ -0,0 +1,41 @@
+import { createProjectLogger, Hex } from '@metamask/utils';
+import { hexToDecimal } from '../../../../shared/modules/conversion.utils';
+import { getNetworkData, getNetworkDataByChainId } from './sentinel-api';
+
+const log = createProjectLogger('transaction-send-bundle');
+
+/**
+ * Returns true if this chain supports sendBundle feature.
+ *
+ * @param chainId - The chain ID to check.
+ * @returns A promise that resolves to true if sendBundle is supported, false otherwise.
+ */
+export async function isSendBundleSupported(chainId: Hex): Promise<boolean> {
+  const network = await getNetworkDataByChainId(chainId);
+
+  if (!network?.sendBundle) {
+    log('Chain is not supported', chainId);
```

Comment:
As this is just the util, should we log from the caller as they can deem if it's log worthy?